.gitmodules

```diff
@@ -1,3 +1,5 @@
 [submodule "jsdoc-template"]
 	path = jsdoc-template
-	url = https://github.com/andreGarvin/jsdoc-template
+	url = git@github.com:openfin/jsdoc-template.git
```

Comment:
Auth problems exist with the https route.

The new core does this better:
https://github.com/HadoukenIO/core/blob/develop/package.json#L22